gradle.properties

```diff
@@ -1 +1,4 @@
-warMode=complete
\ No newline at end of file
+warMode=complete
+aggregateInstanceName=aggregate
+gaeAppId=aggregate
+gaeEmail=some.email@example.org
```

Comment:
I forgot to add the defaults in the properties file! Adding them now...

Reply:
You should also add `aggregateUsername=admininstrator`